src/Browsershot.php

```diff
@@ -940,13 +949,11 @@ protected function getFullCommand(array $command)
 
         if ($this->isWindows()) {
             $fullCommand =
-                $nodeBinary.' '
-                .escapeshellarg($binPath).' '
-                .'"'
-                .$optionsCommand
-                .'"';
+                $this->escapeshellarg($nodeBinary).' '
+                .$this->escapeshellarg($binPath).' '
+                .$optionsCommand;
 
-            return escapeshellcmd($fullCommand);
+            return $fullCommand;
```

Comment:
Specifically its the change to this line here - `escapeshellcmd` ensures that html tags such as `<` and `>` are properly escaped. If I revert just this line here then everything continues to work same as before. What was the reason for removing this @EmanueleCoppola ?

Reply:
Upon further testing it looks like `escapeshellcmd` can make a mess of the command particularly for more complex page headers and can cause json parsing to break inside browser.cjs.

I have side-stepped my issue altogether by simply using `writeOptionsToFile` for the time being.

Reply:
After looking into it more, I realized that there are several problems when dealing with Windows escaping. One of the issues is that it doesn't work well with spaces in the paths of executable files, like "C:\Program Files\nodejs\node.exe".

To solve this problem, the best approach could be to utilize the actual Symfony Process command escaping and let it handle the escaping of paths and commands.

Reply:
Yep agreed!